muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/forum/events/ForumThreadMessageSent.java

```diff
@@ -0,0 +1,30 @@
+package fi.otavanopisto.muikku.plugins.forum.events;
+
+public class ForumThreadMessageSent {
+
+  public ForumThreadMessageSent() {
+  }
+  
+  public ForumThreadMessageSent(Long threadId, Long recipientUserEntityId, String baseUrl) {
+    super();
+    this.threadId = threadId;
+    this.recipientUserEntityId = recipientUserEntityId;
+    this.baseUrl = baseUrl;
+  }
+
+  public Long getThreadId() {
+    return threadId;
+  }
+
+  public Long getRecipientUserEntityId() {
+    return recipientUserEntityId;
+  }
+
+  public String getBaseUrl() {
+    return baseUrl;
+  }
+
+  private Long threadId;
+  private Long recipientUserEntityId;
```

Comment:
Would attribute name `posterUserEntityId` be more accurate here, i.e. the person who posted a new message to the thread.